Libraries/PeriphDrivers/Source/SPIXF/spixf_reva.c

```diff
@@ -83,6 +84,11 @@ static int SPIXFC_ReadRXFIFO(mxc_spixfc_reva_regs_t *spixfc, mxc_spixfc_fifo_rev
 static void SPIXFC_TransHandler(mxc_spixfc_reva_regs_t *spixfc,
                                 mxc_spixfc_fifo_reva_regs_t *spixfc_fifo, mxc_spixf_req_t *req);
 
+#if defined(SPIXF_RAM)
+    int MXC_GetLock_SPIXF(uint32_t *lock, uint32_t value);
+    void MXC_FreeLock_SPIXF(uint32_t *lock);
```

Comment:
Same here as the GPIO naming. Make these static if they're specific to this library only.

Reply:
Fixed.